packages/opentelemetry-instrumentation-grpc/test/helper.ts

```diff
@@ -17,9 +17,7 @@
 import {
   context,
   SpanKind,
-  propagation,
-  setSpan,
-  getSpan,
+  propagation, trace,
```

Comment:
Seems some lint setting has change recently. Same on a few other places.

I don't care much about this so feel free to keep.